test/integration/targets/docker_swarm/tasks/run-test.yml

```diff
@@ -1,8 +0,0 @@
----
-- name: Make sure we're not already using Docker swarm
-  docker_swarm:
-    state: absent
-    force: true
-  diff: no
-- name: "Loading tasks from {{ item }}"
-  include_tasks: "{{ item }}"
```

Comment:
This include was here so that the `tasks included from tasks/tests/xxx.yml` will be displayed before the tasks from that file are executed, and not only during the include loop in `main.yml`. Makes it easier to find the file where the tests are contained in when looking at the output.

Reply:
Ok. I can add a task that displays that information.

Reply:
I tend to leave myself hints in the `name: ` field for each task in a task file, e.g., `DEBIAN |` or `CLEANUP |`. But that's just me.